### PR TITLE
Persistent filter

### LINKED ISSUE #251: Browse - select tag: no way to cancel selection or select "All tags" to clear filter
When clicking select tag, there is no way to cancel and go back to the browse view without making a selection. More importantly, there should be a way to clear the filter, by tapping "All tags" is indicated [here](https://zpl.io/2uqUui) (although we should make the "All" option stand apart more).

### PR DESCRIPTION
**Pull request by JMurphyWeb**
Adds an option to remove filters from a browse view 

resolves #251

### CODE REVIEW

**Comment by nelsonic** [comment]
@JMurphyWeb please `git merge master` into this branch then re-assign to me. thanks! 👍

**Comment by nelsonic** [comment]
👀

File: Procfile [hunk 1]
```diff
@@ -0,0 +1 @@
+web: npm run start && node tags/storeTags.js
```
Comment (nelsonic) [inline Procfile]: does the _order_ of these commands _matter_? :thinking:

File: package.json [hunk 2]
```diff
@@ -9,8 +9,7 @@
     "test": "istanbul cover tape ./test/*/*.test.js | node_modules/tap-spec/bin/cmd.js",
     "coverage": "istanbul check-coverage --statements 100 --functions 100 --lines 100 --branches 100",
     "fast": "tape ./test/*/*.test.js",
-    "generate-tags": "node tags/storeTags.js",
-    "heroku-postbuild": "npm run generate-tags"
+    "generate-tags": "node tags/storeTags.js"
```
Comment (nelsonic) [inline package.json]: does `npm run generate-tags` get run _every_ time you `npm start` ?

**Comment by nelsonic** [comment]
@JMurphyWeb code looks good. 👍 
❤️  ✅  :shipit:

File: package.json (hunk 2 above)
Comment (JMurphyWeb) [inline package.json]: I think it gets run every npm start.

**Pull request merged**